internal/noderesourcetopology/tostring_test.go

```diff
@@ -174,9 +174,11 @@ func TestToString(t *testing.T) {
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "test-nrt",
 				},
-				TopologyPolicies: []string{
-					"restricted",
-					"ignored-from-the-second-onwards",
+				Attributes: []nrtv1alpha2.AttributeInfo{
+					{
+						Name:  TopologyPoliciesAttribute,
+						Value: "restricted,ignored-from-the-second-onwards",
+					},
```

Comment:
even though this is not formally regulated, attributes are commonly key/value pairs. It's legal for the value to be a comma-separated list, but is out of the conventions. Most notably Having a list of values was the expected use case for `topologyPolicies` but is not relevant for attributes. I'd just remove this test.